releasenotes/notes/protein-folding-result-b344ac3c7f48e3ca.yaml

```
@@ -0,0 +1,34 @@
+---
+features:
+  - |
+    Added: :class:`~qiskit_nature.results.ProteinFoldingResult`.
+    This class handles the result found by running a VQE in
+    :class:`~qiskit_nature.problems.sampling.protein_folding.ProteinFoldingProblem`.
+    This class has:
+
+    * Method :meth:`~qiskit_nature.results.ProteinFoldingResult.get_figure` which
+      generates a figure of the shape of the protein.
+
+    * Method :meth:`~qiskit_nature.results.ProteinFoldingResult.save_xyz_file` which
+      saves a .xyz file with the cartesian coordinates of the shape of the molecule.
+
+    * Property :attr:`~qiskit_nature.results.ProteinFoldingResult.protein_shape_file_gen`
+      which is an instance of :class:`~qiskit_nature.results.utils.ProteinShapeFileGen`.
+      This class has in its turn a method
+      :meth:`~qiskit_nature.results.utils.ProteinShapeFileGen.get_xyz_file` which returns
+      an array with the data contained in the .xyz file without having to generate such a file.
+
+    * Property :attr:`~qiskit_nature.results.ProteinFoldingResult.protein_shape_decoder`
+      which is an instance of :class:`~qiskit_nature.results.utils.ProteinShapeDecoder`.
+      This class has in its turn two properties :meth:`~qiskit_nature.results.utils.ProteinShapeDecoder.main_turns`
+      and :meth:`~qiskit_nature.results.utils.ProteinShapeDecoder.side_turns` that return
```

```suggestion
      This class has in its turn two properties :attr:`~qiskit_nature.results.utils.ProteinShapeDecoder.main_turns`
      and :attr:`~qiskit_nature.results.utils.ProteinShapeDecoder.side_turns` that return
```
When properties we should use attr, otherwise though they link, they are rendered with () as though they need to be called as a method